src/sentry/models/organization.py

```diff
@@ -202,9 +203,11 @@ def save(self, *args, **kwargs):
 
         if SENTRY_USE_SNOWFLAKE:
             snowflake_redis_key = "organization_snowflake_key"
+            metrics.incr("organization_with_snowflake_id_create_attempted", sample_rate=1)
             self.save_with_snowflake_id(
                 snowflake_redis_key, lambda: super(Organization, self).save(*args, **kwargs)
             )
+            metrics.incr("organization_with_snowflake_id_created", sample_rate=1)
```

Comment:
nit: Could this be in `save_with_snowflake_id` to reduce repetition?

Reply:
i feel like keeping it outside of `save_with_snowflake_id` distinguishes whether an organization or project is created. if its inside we would have to figure out whats being created with the `snowflake_redis_key`